docs/concepts/ksql-architecture.rst

```diff
@@ -0,0 +1,368 @@
+.. _ksql-architecture:
+
+KSQL Architecture
+#################
+
+The architecture of KSQL enables you to build real-time streaming applications
+from Kafka topics by using only SQL statements and queries. KSQL is built on
+Kafka Streams, so a KSQL application runs on a  Kafka cluster like any other

```

Comment:
"so a KSQL application runs on a  Kafka cluster like any other Kafka Streams application."

NOOOOOOO

A key misconception that people have is that KSQL runs inside the Kafka cluster.  It does not!  And KStreams apps do not either.

Reply:
Thanks for the catch! Is it correct to say that KSQL runs on a cluster of KSQL servers? Or maybe: a KSQL application _communicates_ with a Kafka cluster?

Reply:
@gwenshap called them a "flock" of KSQL servers ;)